cdap-app-fabric/src/test/java/co/cask/cdap/internal/app/namespace/StorageProviderNamespaceAdminTest.java

```diff
@@ -76,5 +84,31 @@ public void test() throws Exception {
     Assert.assertTrue(namespaceLocation.exists());
     storageProviderNamespaceAdmin.delete(myspace);
     Assert.assertFalse(namespaceLocation.exists());
+
+    // test for custom location mapped namespace
+    myspace = new NamespaceId("custom");
```

Comment:
Let's move this to its own test. Seems like separate functionality, and the cost of setup for these tests is not much, since we don't even deploy an application here.